ports/curl/portfile.cmake

```diff
@@ -52,18 +48,30 @@ set(BUILD_OPTIONS
     -DCURL_DISABLE_POP3=ON
     -DCURL_DISABLE_PROXY=OFF
     -DCURL_DISABLE_RTSP=ON
+    -DCURL_DISABLE_SMB=ON
     -DCURL_DISABLE_SMTP=ON
     -DCURL_DISABLE_TELNET=ON
     -DCURL_DISABLE_TFTP=ON
+    -DCURL_ENABLE_MQTT=OFF
     # ENABLE options
     -DENABLE_ARES=OFF
     -DENABLE_MANUAL=OFF
     -DENABLE_THREADED_RESOLVER=ON
+    -DENABLE_UNIX_SOCKETS=OFF
     # USE options
     -DUSE_NGHTTP2=ON
     -DUSE_WIN32_LDAP=OFF
 )
 
+# Check for ca-bundle feature
+if (ca-bundle IN_LIST FEATURES)

```

Comment:
Is none a reasonable default for the CA bundle support? It seems a bit strange as a default to me.

Reply:
I'm kinda torn here cause I could see a couple different scenarios.

There's the ability to set the [default features in a portfile]( https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/control-files.md#default-features) so my initial thought was to add it there. But curl by default will try and use the system SSL so would it be proper to use curl's ca bundle in that case? Also even if it were on as a default feature we specify the features for our setup so that default would end up getting ignored.

I don't really have a strong preference on this.

Reply:
Hmm, the by default using system SSL seems like a reasonable argument that this is okay to not use the curl ca bundle. I was worried about it generating something non-functional if someone did a direct vcpkg installation.